star-ros2/src/star_simulation/test/test_heartbeat_loss.py

```
@@ -0,0 +1,151 @@
+    @classmethod
+    def setUpClass(cls):
+        rclpy.init()
+        cls.node = Node('test_heartbeat_loss')
+        cls.diag_pub = cls.node.create_publisher(
+            DiagnosticArray, '/diagnostics', 10)
+        cls.estop_active = False
+        cls.estop_sub = cls.node.create_subscription(
+            Bool, '/emergency_stop', cls._estop_cb, 10)
+
+    @classmethod
+    def _estop_cb(cls, msg):
```

Ruff (0.15.9)

[warning] Missing return type annotation for classmethod `_estop_cb`

Add return type annotation: `None`

(ANN206)